.gitignore

```diff
@@ -7,6 +7,7 @@
 
 .deps
 Makefile.in
+.cache
```

Comment:
changes to this file are unrelated to any other PR and I think this overlaps with #76; please have a look there